src/test/java/core/basesyntax/dao/DataReaderTest.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.exceptions.NullFileNameException;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class DataReaderTest {
+    private static DataReader dataReader;
+
+    @BeforeClass
+    public static void beforeClass() {
+        dataReader = new FruitFileReader();
+    }
+
+    @Test(expected = NullFileNameException.class)
+    public void read_NullFileName_NotOk() {
+        dataReader.readData(null);
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void read_UnfoundedFile_NotOk() {
+        dataReader.readData("src\\main\\resources\\tasty.csv");
+    }
+
+    // This test can be built during mvn clean package,
+    // but it doesn't pass through GitHub tests, and I don't know why
+    /*
+    @Test
+    public void read_BasicFile_Ok() {
+        List<String> transactionList = dataReader.readData("src\\main\\resources\\test.csv");
+        int expectedTransactionListSize = 8;
+        Assert.assertEquals(expectedTransactionListSize, transactionList.size());
+    }
+     */
```

Comment:
This is because program cant's find this file. Try to use `File.separator` instead of `\\` in your file paths